.github/workflows/release.yml

```diff
@@ -0,0 +1,219 @@
+name: Release
```

Comment:
release file seems old? it has `node 16` and uses old actions and `set-output`

Reply:
have not updated workflows just yet.